.pep8

```diff
@@ -0,0 +1,3 @@
+[flake8]
```

Comment:
why this is needed? The project use `setup.cfg` for flake8 configuration, but this should be already addressed there

Reply:
Yeah good point, can probably remove this.